src/main/java/Bob/Bob.java

```diff
@@ -0,0 +1,167 @@
+//package Bob;
+//
+//import Bob.Storage.Storage;
+//import Bob.TaskList.TaskList;
+//import Bob.Ui.Ui;
+//import Bob.Parser.Parser;
+//import Bob.Command.Command;
+//import Bob.Exception.BobException;
+//
+//import java.util.Scanner;
+//
+///**
```

Comment:
Any reason why you kept this dead code in the file? I see this in some other places as well.